src/xdocs/config_coding.xml

```diff
@@ -4303,6 +4309,17 @@ for(int i=0; i&lt;10; i++) {
   &lt;property name=&quot;max&quot; value=&quot;3&quot;/&gt;
 &lt;/module&gt;
         </source>
+        <p>Code Example:</p>
+        <source>
+if (x > 2) {
+   if (x > 3) {
+      if (x > 4) {
+         if (x > 5) {} // OK till here
+         else {} // violation, depth exceeded '3'
```

Comment:
same here, no violation.
Actually you could check the violation with checkstyle cli with  
```
java -jar checkstyle-8.38-all.jar -c config.xml MyClass.java
```
See here for more info: https://checkstyle.org/cmdline.html#Download_and_Run

Reply:
resolved